tests/x11regressions/x11regressions_setup.pm

```diff
@@ -14,19 +14,14 @@ use testapi;
 sub run() {
     my $self = shift;
 
-    x11_start_program("xterm");
-
-    # grant permission for default user to access serial port
-    type_string "xdg-su -c 'chown $username /dev/$serialdev'\n";
-    wait_still_screen;
+    # cmd1 grant user permission to access serial port until next reboot
+    # cmd2 get permanent user permission to access serial port even if reboot
+    my $cmd1 = "chown $username /dev/$serialdev";
+    my $cmd2 = "gpasswd -a $username dialout";
 
-    if ($password) {
-        type_password;
-        send_key "ret";
-    }
+    x11_start_program("xterm");
 
-    wait_still_screen;
-    save_screenshot;
+    script_sudo "$cmd1; $cmd2";
```

Comment:
better split this in 2 lines. I wonder anyway what's the benefit of having this variables set.

The downside of the ; is that you never know what script_sudo does with the argument. This won't do what you intend:
 > sudo $cmd1; $cmd2